src/System.Diagnostics.DiagnosticSource/tests/Configurations.props

```diff
@@ -2,11 +2,8 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
-      netstandard1.1;
```

Comment:
We got failures trying to build these test configurations because of the way we build with xunit.   After some discussion with @ericstj we concluded that these configurations did not make sense (it did not increase coverage over the other configurations that are still present).   I can forward the e-mail conversation on it if you wish.